src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
```

Comment:
There is an unnecessary empty line after the class declaration (line 2). According to the checklist, we should not add redundant empty lines after class declaration.